bom/pom.xml

```diff
@@ -96,7 +96,7 @@
       <dependency>
         <groupId>org.idpf</groupId>
         <artifactId>epubcheck</artifactId>
-        <version>4.0.1</version>
+        <version>4.0.3-SNAPSHOT</version>
```

Comment:
epubcheck v4.1.0 will be released very soon (no later than 3d week of November), maybe we can wait for it and include it in this PR?

Reply:
OK. I'll make a PR from https://github.com/w3c/epubcheck/compare/master...daisy:super/dependencies then.